src/pages/ProductPage/index.scss

```
@@ -1,14 +1,65 @@
-.product-page{
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  width: 700px;
+.product-page {
+  display: grid;
+  grid-template-columns: 400px 600px;
+  grid-column-gap: 100px;
+  max-width: 1000px;
+  margin: 0 auto;
 
-  .product-info{
+  #overlayOn .gallery {
+      .button.button__icon--previous {
+        z-index: 999;
+        top: 45%;        
+        left: 35%;        
+      }
+    
+      .button.button__icon--next {
+        z-index: 999;
+        top: 45%;
+        left: 63%;
+      }
+  }

```

use 2 space of indentation:

```suggestion
  #overlayOn .gallery {
    .button.button__icon--previous {
      z-index: 999;
      top: 45%;        
      left: 35%;        
    }
    
    .button.button__icon--next {
      z-index: 999;
      top: 45%;
      left: 63%;
    }
  }
```